tests/libvirt/models/compute/server_tests.rb

```diff
@@ -79,5 +82,45 @@
       end
       test("with q35 machine type on x86_64") { server.to_xml.match?(%r{<type arch="x86_64" machine="q35">hvm</type>}) }
     end
+    test("with efi firmware") do
+      server = Fog::Libvirt::Compute::Server.new(
+        {
+          :os_firmware => "efi",
+          :os_firmware_features => {
+            "secure-boot" => "no",
+            "enrolled-keys" => "no"
+          },
+          :nics => [],
+          :volumes => []
+        }
+      )
+      xml = server.to_xml
+
+      os_firmware = xml.include?('<os firmware="efi">')
+      secure_boot = !xml.include?('<feature name="secure-boot" enabled="no" />')
+      enrolled_keys = !xml.include?('<feature name="enrolled-keys" enabled="no" />')
```

Comment:
I would expect these features to show up.
```suggestion
      secure_boot = xml.include?('<feature name="secure-boot" enabled="no" />')
      enrolled_keys = xml.include?('<feature name="enrolled-keys" enabled="no" />')
```

If they're not expected, I'd use a much broader matcher just to make sure it's not some XML formatting:
```suggestion
      secure_boot = !xml.include?('secure-boot')
      enrolled_keys = !xml.include?('enrolled-keys')
```

Reply:
Yeah, my bad. I forgot to fix this.